.kokoro/continuous/notebook.cfg

```diff
@@ -13,7 +13,7 @@ env_vars: {
 
 env_vars: {
     key: "GOOGLE_CLOUD_PROJECT"
-    value: "bigframes-load-testing"
+    value: "bigframes-benchmarking"
```

Comment:
+@shobsi, who wants to modify the GCS project for notebook tests: https://github.com/googleapis/python-bigquery-dataframes/pull/976/files

Reply:
Removed change for this file for now, latter will remove benchmarking for streaming then  change project.